blocks/api/test/factory.js

```diff
@@ -144,7 +144,7 @@ describe( 'block factory', () => {
 				value: 'ribs',
 			} );
 
-			const transformedBlocks = switchToBlockType( block, 'core/updated-text-block' );
+			const transformedBlocks = switchToBlockType( [ block ], 'core/updated-text-block' );
```

Comment:
To avoid all these changes, we could have used lodash's `castArray` in the `switchToBlockType` function